src/app/core/store/users/users.effects.ts

```diff
@@ -83,6 +83,22 @@ export class UsersEffects {
     )
   );
 
+  @Effect()
+  public getCurrentUserWithLastLogin$: Observable<Action> = this.actions$.pipe(
+    ofType<UsersAction.GetCurrentUserWithLastLogin>(UsersActionType.GET_CURRENT_USER_WITH_LAST_LOGIN),
+    tap(() => this.store$.dispatch(new UsersAction.SetPending({pending: true}))),
+    mergeMap(() =>
+      this.userService.getCurrentUserWithLastLogin().pipe(
+        map(user => convertUserDtoToModel(user)),
+        mergeMap(user => [new UsersAction.GetCurrentUserSuccess({user}), new UsersAction.SetPending({pending: false})]),
```

Comment:
We use `flatMap` when we return multiple actions, but maybe `mergeMap` is also suitable..